### PR TITLE
fixing the images of the bedges which are broken

### LINKED ISSUE #2583: [Contributing] Broken badge images in Contributing.md
**Description**
The images of the badges are broken in the contributing.md 
Screenshot 2022-02-04 at 3 17 39 PM

**Expected Behavior**
They shouldn't be broken

**Screenshots**

**Environment:**
- Host OS: 
- Browser: 

---

[Optional] **Additional Context**

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by AbhishekJamhoriya**
Signed-off-by: AbhishekJamhoriya <jamhoriya.1@iitj.ac.in>

**Description**

This PR fixes #2583

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 7fd9056e7c0a042f164afacf3c29c70a5b2284a7 at: https://6200c5d09586842e6a9ffefe--layer5ng.netlify.app

**Review by Abhijay007 — APPROVED** [review]
LGTM 👍 

PS: not sure about the landscape logo

**Comment by Abhijay007** [comment]
> LGTM 👍

PS: not sure about the landscape logo

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -77,14 +77,14 @@ Badges are a great way of highlighting the area of contribution by any given com
...
-- <img src="https://github.com/layer5io/layer5/blob/master/src/sections/Community/Member-single/landscape-green.png" width="25px" height="25px"/> Landscape
-- <img src="https://github.com/layer5io/layer5/blob/master/src/sections/Community/Member-single/layer5-image-hub.svg" width="25px" height="25px"/> ImageHub
-- <img src="https://github.com/layer5io/layer5/blob/master/src/sections/Community/Member-single/meshery-logo-light.svg" width="25px" height="25px"/> Meshery
-- <img src="https://github.com/layer5io/layer5/blob/master/src/sections/Community/Member-single/meshery-operator-dark.svg" width="25px" height="25px"/> Meshery Operator
-- <img src="https://github.com/layer5io/layer5/blob/master/src/sections/Community/Member-single/smp-dark-text.svg" width="25px" height="25px"/> SMP
+- <img src="https://github.com/layer5io/layer5/blob/master/src/assets/images/community/Layer5-mentor-program.png" width="25px" height="25px"/> Community
+- <img src="https://github.com/layer5io/layer5/blob/master/src/assets/images/landscape/comparison-of-service-mesh-strengths-dark.png" width="25px" height="25px"/> Landscape
```
Comment (warunicorn19) [inline CONTRIBUTING.md:81]: https://github.com/layer5io/layer5/blob/master/src/assets/images/landscape/layer5_landscape_green.svg use this for landscape

**Review by Abhijay007 — APPROVED** [review]
LGTM 👍 

The Icon of SMP seems very small as compared to other icons but I think it is associated with the width of the original image so maybe editing that wouldn't affect the size or may bring some size issues.

**Comment by l5io** [comment]
🚀 Preview for commit 57f0cf9fd939237144c10ea285446f26ff2ef11d at: https://62018189da905f5db0ecae03--layer5ng.netlify.app

**Review by warunicorn19 — APPROVED** [review]
Looks good.

**Pull request merged**